gradle.properties

```diff
@@ -65,7 +65,7 @@ versioningPluginVersion=1.1.0
 # will be downloaded and used. If not set, the existing local installations will be use
 # The version of npm corresponds to the given version of node
 npmVersion=6.14.13
-nodeVersion=14.17.1
+nodeVersion=16.13.1
```

Comment:
Why are we doing this against 21.7? Is a client developer using an M1 Macbook? Can they just change it locally? 

This seems much better to go into `develop` and let it bake rather than destabilizing a release or downstream releases (i.e. v21.11).